src/app/services/views/ServiceListView.vue

```diff
@@ -21,11 +22,19 @@
           <template #title>
             <h2>
               <RouteTitle
+                v-if="route.params.service"
+                :title="t('services.routes.items.tray_title', { name: route.params.service })"
+                :render="true"
+              />
+
+              <RouteTitle
+                v-else
```

Comment:
Why are we change this title based on which service you have in the tray. Its the header for the list so should stay as 'Services'

Reply:
I already explained this here: https://github.com/kumahq/kuma-gui/pull/1611#discussion_r1365099922

Could you please focus on reviewing https://github.com/kumahq/kuma-gui/pull/1613 and not the details of the concrete service tray implementation that are completely unrelated to it?

Reply:
I think this is related to the fact that  we aren't using RouteView and RouteTitle to set the title in the Summary route and trying to do it in the list route instead.

Reply:
As I mentioned offline, #1613 is just a wrapper around KSlideout, there's not a lot to review without seeing it in context. I've left some more comment over there also, but honestly I think we should just close #1613, the code in it is here also, plus all of the additional code in this PR is much more important as that the stuff we want to roll out over the entire application.

Reply:
> the code in it is here also

The code is only in here also because we are, as you know, not able to create pull requests against other pull requests coming from a fork.

> I think we should just close #1613 

I’m not closing this PR because it landing will allow us to continue implementing summary trays in the application. It also makes PR review simpler to work in smaller chunks.

Reply:
Yeah I getcha, sorry I'm not complaining about the code being here, in fact is easier for me to review here as its an implementation plus a single example of how we would use it, which in this case is very important.

> I’m not closing this PR because it landing will allow us to continue implementing summary trays in the application. It also makes PR review simpler to work in smaller chunks.

Ok, well the most important thing for me is how we are to use the tray, I don't really mind too much the innards of it as we can always change that in one place. Feel free to leave that one open, but theres already a lot of conversation here (I think you directed me to here a while back), so if its ok with you, considering all the code is here also, can we keep the reviewing in this PR?

Reply:
> Ok, well the most important thing for me is how we are to use the tray, I don't really mind too much the innards of it as we can always change that in one place. Feel free to leave that one open, but theres already a lot of conversation here (I think you directed me to here a while back), so if its ok with you, considering all the code is here also, can we keep the reviewing in this PR?

I’m sorry, but no, it isn’t. I’ve been quite clear about what’s up for review and it’s not this PR but #1613. This is indicated by this PR being in draft state and #1613 not being in draft state. It’s also clear from me specifically asking about the review status of #1613 on a few occasions.

I _don’t_ want this PR to be reviewed first because that defeats the purpose of working in smaller chunks of changes. It’s important to see how #1613 is used and that’s why this PR is available, but pretty much all review comments on this PR are now about it and not #1613.

So, can I please ask you to review #1613 so we can both unblock the following summary tray implementation tasks _and_ consider this PR for what it is?

Reply:
I've added similar concerns over on #1613